tests/infra_test.go

```diff
@@ -35,9 +36,11 @@ import (
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	gomegatypes "github.com/onsi/gomega/types"
+	errs "github.com/pkg/errors"
```

Comment:
Why is this needed?

Reply:
`if errs.As(err, &e) {`

Reply:
You do not need it, it is in the stdlib.